installer/cli/uninstallcli.py

```diff
@@ -0,0 +1,33 @@
+import click
+from installer.helpers.destroyprep import destroyprep
+from installer.helpers.terraform import exec_terraform_destroy, get_terraform_output_var
+
+
+@click.command()
+@click.option(
+    '--mode',
+    type=click.Choice(['all', 'frameworkonly']),
+    help='`all` to remove Jazz and all deployed functions, \
+    `frameworkonly` to remove Jazz but leave deployed functions alone',
```

Comment:
Same comment as above - "all deployed services" as we have API's, Fns and SPAs.

Reply:
Fixed.